lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,49 @@
+<div class="cohort-discussions-nav">
+    <form action="" method="post" id="cohort-coursewide-discussions-form" class="cohort-coursewide-discussions-form">
+        <h3 class="subsection-title"><%- gettext('Designate discussion topics as divided by cohort') %></h3>
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="coursewide-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Coursewide Discussion Topics') %></h3>
+                        <p><%- gettext('You can optionally designate coursewide discussions topics to be cohorted.') %></p>
```

Comment:
If it's in keeping with what we do in other areas, I would like to suggest making this a more active instruction: "Select the course-wide discussion topics that you want to divide by cohort." @explorerleslie @marcotuts @muzaffaryousaf